### ISSUE TITLE
4D biome sampling (3D x 1D Noise)

### ISSUE THREAD

**Issue by inayakitori** [inayakitori/4dmc#24]
So for this biome sampling, the 5D point of temperature, humidity, continentalness, erosion, and weirdness should be taken from the block's (x,y,z) position but then another sample should be taken depending on the W position and those two should be blended sogether in some way. This will lead to slices roughly keeping similar biomes but some slices will have more temperature or maybe more weirdness, allowing from each world slice to have it's own unique feel to it.

**Comment by Frisk-256** [comment]
why not use 4d noise?

**Comment by Enator18** [comment]
Yeah why can't you use 4D noise? That would be very cool and probably more consistent. Could also work for terrain as well.
